src/test/java/uk/gov/pay/api/it/CachingAuthenticatorTest.java

```diff
@@ -99,6 +99,7 @@ private String aPayment() {
         JsonStringBuilder jsonStringBuilder = new JsonStringBuilder()
                 .add("charge_id", "chargeId")
                 .add("amount", 100)
+                .add("language", "en")
```

Comment:
Is this part of the intended change?

Reply:
Yes, please see commit https://github.com/alphagov/pay-publicapi/pull/403/commits/622b968b207c01097431ca3e49c37d26e3591a2a for details